apps/public-docsite-v9/package.json

```diff
@@ -31,9 +34,6 @@
     "@fluentui/react-northstar": "*",
     "@fluentui/react-icons-northstar": "*",
     "@fluentui/react-components": "*",
-    "@fluentui/react-storybook-addon": "*",
```

Comment:
hmm so my approach here is, anything that is in runtime for `type:application` should be a production dependency.

While we those addons are added during "dev"(build) time, they are part of runtime. I think we should revert this change.

Reply:
Then their stories will appear in the list because of:

https://github.com/microsoft/fluentui/blob/f357c800c5f39bfe46a9081ce47f38b983dfbe7e/apps/public-docsite-v9/.storybook/main.js#L15

https://github.com/microsoft/fluentui/blob/f357c800c5f39bfe46a9081ce47f38b983dfbe7e/scripts/storybook/src/utils.js#L225

@Hotell other ideas?

Reply:
once this lands https://github.com/microsoft/fluentui/pull/30844 lets use that new api to exclude those problematic packages - so please undo these package.json changes